### PR TITLE
Updated readme with tool arguments

### PR DESCRIPTION
**Pull request by ryangrundy7**
I've updated the README just to show the arguments that can be used with the various tools that are in the repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,20 @@ The most bestestest tools for make running the census big success wow.
...
 ## How to use - Message Manipulator
 
+The message manipulator will allow you to look at rabbit messages inside a queue you specify.
+### Arguments
+
+| Name                      | Description                                                                                                         |                                                                                        
+| ---------------------     | ------------------------------------------------------------------------------------------------------------------- |
+| `source_queue_name`       | Rabbit queue to find messages on                                                                                    |
+| `message_hash_search`     | Rabbit message hash to search by                                                                                    |
+| `-s --search`             | Search for a string inside of the rabbit message body                                                               |
```
Comment (liamtoozer) [inline README.md:14]: Is it worth mentioning the `-l --limit` flags here as well? I guess the default value is fine for most situations though...

**Review by liamtoozer — APPROVED** [review]
LGTM 👍

File: README.md [hunk 2]
```diff
@@ -64,21 +91,39 @@ Delete message on pubsub subscription:
...
+
+
 
 Moving a pubsub message to a bucket:
 ```bash
 python put_message_on_bucket.py <subscription name> <subscription project id> <bucket name> <message_id>
 ```
 ## How to use - publishing message from GCS bucket to pubsub topic
 
+Move messages from pubsub to a GCS bucket
```
Comment (liamtoozer) [inline README.md]: The heading of this section doesn't match this description by the looks of it?

File: README.md [hunk 3]
```diff
@@ -64,21 +91,39 @@ Delete message on pubsub subscription:
...
+### Arguments
+
+| Name                      | Description                                                                                                         |                                                                                        
+| ---------------------     | ------------------------------------------------------------------------------------------------------------------- |
+| `topic name`              | Subscription name to look on                                                                                        |
```
Comment (liamtoozer) [inline README.md]: `Subscription` -> `Topic`?

File: README.md (hunk 1 above)
Comment (ryangrundy7) [reply README.md:14]: I've left it out because in Adams recent PR that was merged, it was removed and hardcoded to 100

Comment (liamtoozer) [reply README.md:14]: Ah sorry, I don't think I've pulled from this repo in a while!

File: README.md [hunk 4]
```diff
@@ -64,21 +91,39 @@ Delete message on pubsub subscription:
...
+### Arguments
+
+| Name                      | Description                                                                                                         |                                                                                        
+| ---------------------     | ------------------------------------------------------------------------------------------------------------------- |
+| `topic name`              | topic name to put message on                                                                                               |
+| `project id`              | GCP project name                                                                                                    |
+| `bucket name`             | Bucket you want to move the pubsub message to                                                                       |                                                  
+| `bucket blob name`        | Name of the blob you want to publish to a topic                                                                     |                                                  
+
 Moving a pubsub message to a bucket:
```
Comment (gavinleeedwards) [inline README.md]: This heading should be  `Publishing message from GCS bucket to pubsub topic`

**Review by gavinleeedwards — CHANGES_REQUESTED** [review]

**Review by gavinleeedwards — APPROVED** [review]
👍

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

**Review by liamtoozer — APPROVED** [review]

**Review by gavinleeedwards — APPROVED** [review]

**Pull request merged**